test/screenshotter/ss_data.yaml

```diff
@@ -0,0 +1,85 @@
+Accents: \vec{A}\vec{x}\vec x^2\vec{x}_2^2\vec{A}^2\vec{xA}^2
```

Comment:
This looks much better!

How did you generate this? Was it done automatically?

Now that this is in yaml, could you add some comments at the top describing the options that we can have in here?